### ISSUE TITLE
Vertical FieldGroup does nothing when used in Form

### ISSUE THREAD

**Issue by Harbs** [adobe/spectrum-css#484]
## Description
I'm not sure if this a documentation issue or a bug, but the Forms markup has FieldGroups with a vertical variant. The vertical variant does not work in a Form itemField.

## Link to documentation
https://opensource.adobe.com/spectrum-css/components/form/#Form-(labels-left)

## Additional context
In the following markup, you will find two references to `spectrum-FieldGroup--vertical`

````
  <div class="spectrum-Form-item">
    <label class="spectrum-Form-itemLabel spectrum-FieldLabel--left" for="spectrum-textinput-instance">Interests</label>
    <div class="spectrum-Form-itemField">
      <div class="spectrum-FieldGroup spectrum-FieldGroup--vertical">
        <div class="spectrum-Radio">
          <input type="radio" name="animals" class="spectrum-Radio-input" id="radio-5">
          <span class="spectrum-Radio-button"></span>
          <label class="spectrum-Radio-label" for="radio-5">Kittens</label>
        </div>
        <div class="spectrum-Radio">
          <input type="radio" name="animals" class="spectrum-Radio-input" id="radio-6">
          <span class="spectrum-Radio-button"></span>
          <label class="spectrum-Radio-label" for="radio-6">Dogs</label>
        </div>
      </div>

      <div class="spectrum-FieldGroup spectrum-FieldGroup--vertical">
        <label class="spectrum-Checkbox">
          <input type="checkbox" class="spectrum-Checkbox-input" id="checkbox-0">
          <span class="spectrum-Checkbox-box">
            <svg class="spectrum-Icon spectrum-UIIcon-CheckmarkSmall spectrum-Checkbox-checkmark" focusable="false" aria-hidden="true">
              <use xlink:href="#spectrum-css-icon-CheckmarkSmall" />
            </svg>
          </span>
          <span class="spectrum-Checkbox-label">Checkbox</span>
        </label>
        <label class="spectrum-Checkbox">
          <input type="checkbox" class="spectrum-Checkbox-input" id="checkbox-1" checked>
          <span class="spectrum-Checkbox-box">
            <svg class="spectrum-Icon spectrum-UIIcon-CheckmarkSmall spectrum-Checkbox-checkmark" focusable="false" aria-hidden="true">
              <use xlink:href="#spectrum-css-icon-CheckmarkSmall" />
            </svg>
          </span>
          <span class="spectrum-Checkbox-label">Checkbox</span>
        </label>
      </div>
    </div>
  </div>
````

**Comment by lazd** [comment]
@Harbs you're right, that class seems to do nothing. I don't know if this is intentional, we'll have to follow up with design. Thanks!

**Comment by GarthDB** [comment]
Fixed with #897
